File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.java
import java.util.*;

public class Main
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
	    
	    int C = sc.nextInt();
	    
	    for (int i = 0; i < C; i++){
	        int N = sc.nextInt();
	        int[] scores = new int[N];
	        int sum = 0;
	        double cnt = 0;
	        
	        for (int j = 0; j < N; j++) {
	            int score = sc.nextInt();
	            scores[j] = score;
	            sum += score;
	        }
	        
	        double avg = (sum / N);
	        
	        for (int j = 0; j < N; j++) if (scores[j] > avg) cnt++;
	        
	        System.out.printf("%.3f%%\n", 100 * cnt / N);
	    }
	}
}

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.c
#include <stdio.h>

int main()
{
    int C, N;
    scanf("%d", &C);
    
    while (C--) {
        scanf("%d", &N);
        int scores[N], sum = 0, cnt = 0;
        double avg = 0;
        
        for (int i = 0; i < N; i++) {
            scanf("%d", scores + i);
            
            sum += scores[i];
        }
        
        avg = (double) sum / N;
        
        for (int i = 0; i < N; i++) if (scores[i] > avg) cnt++;
        
        printf("%.3lf%%\n", (double) 100 * cnt / N);
    }
}